Commands/InteractionCommands/ClearInteractions.cs

```diff
@@ -0,0 +1,264 @@
+
+            // If all args are unset
+            if (count == 0 && upTo == "" && user == default && ignoreMods == false && match == "" && botsOnly == false && humansOnly == false && imagesOnly == false && linksOnly == false)
+            {
+                await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} You must provide at least one argument! I need to know which messages to delete.").AsEphemeral(true));
+                return;
+            }
+
+            if (count == 0 && upTo == "")
+            {
+                await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} I need to know how many messages to delete! Please provide a value for `count` or `up_to`.").AsEphemeral(true));
+                return;
+            }
+
+            // If count is too low or too high, refuse the request
+
+            if (count < 0)
+            {
+                await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} I can't delete a negative number of messages! Try setting `count` to a positive number.").AsEphemeral(true));
+                return;
+            }
+
+            if (count >= 1000)
+            {
+                await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} Deleting that many messages poses a risk of something disastrous happening, so I'm refusing your request, sorry.").AsEphemeral(true));
+                return;
+            }
+
+            // Get messages to delete, whether that's messages up to a certain one or the last 'x' number of messages.
+
+            if (upTo != "" && count != 0)
+            {
+                await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} You can't provide both a count of messages and a message to delete up to! Please only provide one of the two arguments.").AsEphemeral(true));
+                return;
+            }
+
+            List<DiscordMessage> messagesToClear;
+            if (upTo == "")
+            {
+                var messages = await ctx.Channel.GetMessagesAsync((int)count);
+                messagesToClear = messages.ToList();
+            }
+            else
+            {
+                DiscordMessage message;
+                ulong messageId;
+                if (!upTo.Contains("discord.com"))
+                {
+                    try
+                    {
+                        messageId = Convert.ToUInt64(upTo);
+                    }
+                    catch
+                    {
+                        await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} That doesn't look like a valid message ID or link! Try again."));
+                        return;
+                    }
```

Comment:
Rather than throwing and suppressing an exception, it's cleaner to use `ulong.TryParse` here.
I also added a "Please".
```suggestion
                    if (!ulong.TryParse(upTo, out messageId))
                    {
                        await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().WithContent($"{Program.cfgjson.Emoji.Error} That doesn't look like a valid message ID or link! Please try again."));
                        return;
                    }
```